.cicdtemplate/.github/workflows/deploy_to_firebase_app_distribution_staging_and_production.yml

```diff
@@ -37,43 +40,43 @@ jobs:
       - name: Run Detekt
         run: ./gradlew detekt
 
-      - name: Archive Detekt report
+      - name: Archive Detekt reports
         uses: actions/upload-artifact@v2
         with:
-          name: detekt-report
+          name: detekt-reports
           path: build/reports/detekt/
 
-      - name: Run Unit Tests & Jacoco
+      - name: Run unit tests and Jacoco
         run: ./gradlew jacocoTestReport
 
-      # Archive Unit test & code coverage report
-      - name: Archive Code coverage report
+      # Archive Jacoco code coverage reports
+      - name: Archive code coverage reports
         uses: actions/upload-artifact@v2
         with:
-          name: code-coverage-report
+          name: code-coverage-reports
           path: |
             app/build/reports/jacoco/jacocoTestReport/
             data/build/reports/jacoco/jacocoTestReport/
 
-      # Build Staging Debug
-      - name: Build Staging
+      # Build staging debug
+      - name: Build staging
```

Comment:
"Build staging APK" is clearer. Or sometimes you will have arb as well so need to be clear in here